src/languages/hi.ts

```diff
@@ -14,9 +14,19 @@ export default {
   currencyConversionRate: "मुद्रा रूपांतरण दर",
   volumeConversionRate: "मात्रा रूपांतरण दर",
   convertedCost: "रूपांतरित लागत",
-  volumeFormulaPerBottomUnit: "{topUnitCount} {topUnit}s = 1 {bottomUnit}",
-  volumeFormulaPerTopUnit: "1 {topUnit} = {bottomUnitCount} {bottomUnit}s",
-  gasPriceFormula: "{number} {currency} por {unit}",
+  liter: `{quantity, plural,
+  one {लीटर}
+  other {लीटर}
```

Comment:
No need to have the plural logic here, seems like Hindi doesn't use it for liters and gallons like this